events.js

```diff
@@ -0,0 +1,21 @@
+var CarLot = (function () {
+
+// Calls by tag name then adds event listener through array //
+  CarLot.activateEvents = function () {
+    var cars = document.getElementsByTagName("car");
+
+    for (var i = 0; i < cars.length; i++) {
+      cars[i].addEventListener("click", function () {
+        var car = event.currentTarget;
+        var background = "#c6fdf4";
+        CarLot.border(car, background);
+      });
+    }
+
+// also adds keyup attribute //
+    var textInput = document.getElementById("text-input");
+    textInput.setAttribute("onkeyup", "CarLot.replaceDescription()");
+  }
+
+  return CarLot;
+})(CarLot);
```

Comment:
You're passing in an argument to a function that doesn't accept an argument. On line four you need to create an arg for the function to take. Then, as above, you need to augment the object you're passing in, not CarLot directly